src/styles/page.scss

```diff
@@ -0,0 +1,65 @@
+* {
+  box-sizing: border-box;
+  margin: initial;
+  padding: initial;
```

Comment:
do you need it?

Reply:
yes, they effect on the layout (I tried), is it possible to reassign in another place?